### ISSUE TITLE
Logging not working from flask

### ISSUE THREAD

**Issue by carolinebeauchamp** [lyft/metadataproxy#51]
With debug switched on debug output from gunicorn appears in the logs but not output from the application itself, via flask.

something akin to:

```
log = logging.getLogger(__name__)
stream_handler = StreamHandler(stream=sys.stdout)
app.logger.addHandler(stream_handler)
```
in `__init__.py` appears to make it work, along with setting the logging level.

I can do a pull request for this, unless someone knows a better way - python isn't my first language, so I'm not completely sure whether this is the usual/best/conventional way of doing this :)

**Comment by ryan-lane** [comment]
Released in 1.11.0

**Comment by ryan-lane** [comment]
Let me know if things aren't working as intended in that version.
